views/postgres/create.erb

```diff
@@ -174,6 +174,38 @@
                 </div>
               </div>
 
+              <div class="col-span-full">
```

Comment:
Why don't we use `radio-small-cards` component? Does it have any customization?

Reply:
We use it in a child element. Do you want to move it above?